packaging/nuget/package.ps1

```diff
@@ -0,0 +1,26 @@
+Push-Location $PSScriptRoot
+[Environment]::CurrentDirectory = $PWD
+
+$ProjectsDir = "..\..\src"
```

Comment:
@brthor is right, the exit would occur before pop-location, see my comment above.